tests/test_tools.py

```diff
@@ -1,11 +1,12 @@
 #!/usr/bin/env pytest -vs
-"""Tests for Validation functions."""
+"""Tests for GoPhish tool functions."""
 
 # Third-Party Libraries
+from mock import patch
 import pytest
 
 # cisagov Libraries
-from tools.gophish_complete import get_campaign_id
+from tools import gophish_complete, gophish_export
```

Comment:
I'd import just the functions that you plan to use:
```
from tools.gophish_complete import get_campaign_id
from tools.gophish_export import assessment_exists, import_users
```

Reply:
Changed back to importing only the specific functions that are needed in commit 0605cb6.